Cargo.toml

```diff
@@ -9,5 +9,7 @@ members = [
   "palabritas-common",
   "palabritas-compiler",
   "cli",
+  "cli-2",
```

Comment:
We don't need to do this, please remove the previous cli, it's still in github if we need it.